tests/settings.py

```diff
@@ -1,34 +1,47 @@
 import os
 import sys
 
-
+DEBUG = True
 BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+SECRET_KEY = 'cookies_are_delicious_delicacies'
+ROOT_URLCONF = 'urls'
+LANGUAGE_CODE = 'en-us'
+TIME_ZONE = 'UTC'
+USE_I18N = True
+USE_L10N = True
+USE_TZ = True
+MIDDLEWARE_CLASSES = ()
+DEFAULT_INDEX_TABLESPACE = ''
+
 # Make sure the copy of seeker in the directory above this one is used.
 sys.path.insert(0, BASE_DIR)
-SECRET_KEY = 'cookies_are_delicious_delicacies'
+
 INSTALLED_APPS = (
     'bungiesearch',
     'core',
 )
-ROOT_URLCONF = 'urls'
+
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
         'NAME': ':memory:',
     }
 }
-LANGUAGE_CODE = 'en-us'
-TIME_ZONE = 'UTC'
-USE_I18N = True
-USE_L10N = True
-USE_TZ = True
-DEBUG = True
-MIDDLEWARE_CLASSES = ()
-DEFAULT_INDEX_TABLESPACE = ''
-BUNGIESEARCH = {'URLS': [os.getenv('ELASTIC_SEARCH_URL')],
-                'INDICES': {'bungiesearch_demo': 'core.search_indices',
-                            'bungiesearch_demo_bis': 'core.search_indices_bis'},
-                'ALIASES': {'bsearch': 'core.search_aliases'},
-                'SIGNALS': {'BUFFER_SIZE': 1, 
-                            'SIGNAL_CLASS': 'core.bungie_signal.BungieTestSignalProcessor'},
-                'ES_SETTINGS': {'http_auth': os.getenv('ELASTIC_SEARCH_AUTH')}}
+
+BUNGIESEARCH = {
+    'URLS': [os.getenv('ELASTIC_SEARCH_URL', 'localhost')],
+    'ES_SETTINGS': {
+        'http_auth': os.getenv('ELASTIC_SEARCH_AUTH')
+    },
+    'ALIASES': {
+        'bsearch': 'core.search_aliases'
+    },
+    'INDICES': {
+        'bungiesearch_demo': 'core.search_indices',
+        'bungiesearch_demo_bis': 'core.search_indices_bis'
+    },
+    'SIGNALS': {
+        'BUFFER_SIZE': 1,
+        'SIGNAL_CLASS': 'core.bungie_signal.BungieTestSignalProcessor'
+    }
+}
```

Comment:
I just prettied up this file, no changes except the default to ELASTIC_SEARCH_URL. I can revert these changes if you want.